### PR TITLE
Added getMetaData operation

### PR DESCRIPTION
**Pull request by skavanagh**
Added operation to return a hashmap for each set of metadata properties

### CODE REVIEW

**Comment by skavanagh** [comment]
It looks like from the 'meta' table a record either has the contents of the MANIFEST.MF or the pom.properties.  And the only way I see to call `getVulnerabilities(hashMap)`
is to first call`vr.getFlattenedMetaData()` which gives me a HashMap of everything in both the MANIFEST.MF and the pom.propeties, so it fails on matching the record.  

To get around this I added an operation that returns a HashMap for each individual data file.  That way you can call `getVulnerabilities(hashMap)` like so 

```
for (String key : vr.getMetaData().keySet()) {
    HashSet<String> cves = db.getVulnerabilities(vr.getMetaData().get(key));
    if (!cves.isEmpty()) {
        vulnerable = true;
    }
}
```

I'm probably completely missing something here though :smile:

File: src/main/java/com/redhat/victims/VictimsRecord.java [hunk 1]
```diff
@@ -329,6 +329,25 @@ public String getHash(Algorithms alg) {
 		}
 		return result;
 	}
+
```
Comment (abn) [inline src/main/java/com/redhat/victims/VictimsRecord.java]: Formatting issues:
- use tabs (4) not spaces
- truncate all white spaces at the end of lines

File: src/main/java/com/redhat/victims/VictimsRecord.java [hunk 2]
```diff
@@ -329,6 +329,25 @@ public String getHash(Algorithms alg) {
 		}
 		return result;
 	}
+	
+    /**
+     * Returns a key/value map for each individual set of metadata properties
```
Comment (abn) [inline src/main/java/com/redhat/victims/VictimsRecord.java]: Maybe say something along the lines of `Returns a file to properties map for each available {@link MetaRecord}` ?

**Comment by abn** [comment]
Note that currently `setFromMetadata` does not support fetching artifact coordinates from `pom.properties` yet. The overall implementation surrounding properties and metadata might require an overhaul in the future.

**Comment by skavanagh** [comment]
Updated the method doc and I think I have the whitespace issues figured out. Thanks @abn !!!

**Pull request merged**